CONTRIBUTING.md

```diff
@@ -1,13 +1,43 @@
-Help us to make OpenMS better.
-Among other things, you can
-  - implement new features (mostly C++)
+Help us to make OpenMS better and become part of the OpenMS open source community.
+
+#General remarks:
+
+There are many ways how you can contribute without needing to program. For instance, you can:
+  - report issues like bugs
+  - pose questions
+  - point out missing documentation
+  - request new features
   - write/update documentation
-  - fix bugs
-  - ...
-  
-To incorporate your contributions, a pull request against the OpenMS 'develop' branch is probably
-the most efficient way. A core developer will review your changes and approve them (or ask for modifications).
 
-Further details on our GitHub-Wiki: https://github.com/OpenMS/OpenMS/wiki#-for-developers
+If you are a programmer you can open a Pull Request to:
+  - implement new algorithms, features or tools
+  - fix bugs and resolve issues
+
+we are happy to review your contributions and point out potential ways on how to improve them.
+
+#Reporting an issue:
+If you found a bug, e.g. an OpenMS tool crashes during data processing, it is essential to provide some basic information:
+  - the OpenMS version you are running
+  - the platform you are running OpenMS on (Windows 10, ...)
+  - how you installed OpenMS (e.g., from within KNIME, binary installers, self compiled)
+  - a description on how to reproduce the bug
+  - relevant tool output (e.g., error messages)
+  - data to repoduce the bug (If possible as a github gist. Other platforms like dropbox, google drive links also work. If you can't share the data publicly please indicate this and we will contact you in private.)
+
+Optionally:
+  - label your issue using github labels (e.g. as: question, defect) that indicated the type of issue and which components of OpenMS (blue labels) are affected. The severity is usually assigned by OpenMS maintainers and used internally to e.g. indicate if a bug is a blocker for a new release.
+
+#Opening a Pull Request
```

Comment:
user might not know what whis is so
`#Opening a Pull Request (for programmers/developers)`
?